README.md

```diff
@@ -91,3 +66,15 @@ mix phoenix.server
 ```
 
 As live-reloading is enabled, any changes you make to ```src/dguweb/web/static/{css,js}``` or ```src/dguweb/web/elm/*.elm``` or ```src/dguweb/web/templates/*.eex``` should be reflected in the browser without reloading or restarting anything.
+
+## Users
+
+Until authentication is addressed you must perform a manual step to set up your users in the prototype.
+
+1. Connect to your new CKAN from the dgu_deploy repository, and register some users and assign to the appropriate organisations.
+
+2. For each user, visit their user profile and obtain their API Keys.
+
+3. In the dguweb_dev database you need to assign these APIkeys to users who have registered in the prototype front-end.
+
```

Comment:
Maybe it doesn't matter so much for now, but maybe we need to be more specific and name db tables or fields, etc?